### PR TITLE
Export `IndexAlsos` from `psammead_story_promo`

### LINKED ISSUE #1780: Export Index Alsos from `Psammead-story-promo`
**Is your feature request related to a problem? Please describe.**
At the moment we are not exporting the Index Alsos, so we can't import them in Simorgh doing `@bbc/psammead-story-promo/IndexAlsos`.

**Describe the solution you'd like**
We need to export them somehow.

**Describe alternatives you've considered**
 - Export them directly from `psammead-story-promo/src/index.jsx` doing:
    `
    export * as IndexAlsos from './IndexAlsos';
   `

- Create a nested `package.json` like [here](https://github.com/bbc/psammead/tree/latest/packages/utilities/gel-foundations/typography)

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #1780 

**Overall change:** _Export `IndexAlsos` from `psammead_story_promo`._

**Code changes:**

- _Add `@babel/plugin-proposal-export-namespace-from` to babel config._
- _Update `psammead/` CHANGELOG.md and version number._
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by Bopchy** [comment]
I am not sure why the snapshot changed..

**Review by ghost — COMMENTED** [review]
https://github.com/bbc/psammead/issues/1780#issuecomment-523023861

File: packages/components/psammead-story-promo/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-story-promo",
-  "version": "2.5.3",
+  "version": "2.5.4",
```
Comment (dr3) [inline packages/components/psammead-story-promo/package.json]: This isnt a patch as it adds functionality

Comment (Bopchy) [reply packages/components/psammead-story-promo/package.json]: Changed it to a minor change

**Review by DenisHdz — APPROVED** [review]
I thought we were going to go for the `'export * from ./IndexAlsos'` in the end, without using `namespace`, but I am happy with this solution too.

**Comment by Bopchy** [comment]
My bad. I thought we decided on the implemented solution because accessing the different exports would not require `IndexAlsos.[thing being accessed]`

**Review by ghost — APPROVED** [review]

File: packages/components/psammead-story-promo/IndexAlsos/package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@bbc/psammead-story-promo/IndexAlsos",
```
Comment (dr3) [inline packages/components/psammead-story-promo/IndexAlsos/package.json]:
```suggestion
  "name": "@bbc/psammead-story-promo/index-alsos",
```

Comment (dr3) [reply packages/components/psammead-story-promo/IndexAlsos/package.json]: And the folder name too

**Review by dr3 — CHANGES_REQUESTED** [review]
Also can you document this new API in the readme please

**Review by dr3 — APPROVED** [review]

**Pull request merged**